src/index.ts

```diff
@@ -679,6 +679,7 @@ export const formats: IFormat[] = [
   bitcoinBase58Chain('CCA', 489, [[0x0b]], [[0x05]]),
   getConfig('SOL', 501, bs58EncodeNoCheck, bs58DecodeNoCheck),
   bitcoinBase58Chain('LRG', 568, [[0x1e]], [[0x0d]]),
+  hexChecksumChain('TFUEL', 570),
```

Comment:
Is this correct coin type? 570 is allocated to BDX according to https://github.com/satoshilabs/slips/blob/master/slip-0044.md